tests/integration/rdb.tcl

```diff
@@ -6,16 +6,14 @@ set server_path [tmpdir "server.rdb-encoding-test"]
 exec cp tests/assets/encodings.rdb $server_path
 exec cp tests/assets/list-quicklist.rdb $server_path
 
-foreach {type max_listpack_size} {listpack -1 quicklist 0} {
-start_server [list overrides [list "dir" $server_path "dbfilename" "list-quicklist.rdb" save "" "list-max-listpack-size" $max_listpack_size]] {
```

Comment:
why did you give that up?

Reply:
Since the list length in list-quicklist.rdb is 1, it will be a listpack anyway.

Reply:
ohh, right.